### PR TITLE
feat(cspell): use nasa dict

### LINKED ISSUE #6957: [TOOLING]: Use cspell nasa dictionary
#### Summary
Having a dictionary in cspell devoted to nasa is good.

### PR DESCRIPTION
**Pull request by evenstensberg**
#6888

Closes #6957

### Describe your changes:
Uses nasa dictionary instead of declaring grammar in "words" json property.
### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [x] Tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Reviewer has tested changes by following the provided instructions?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate automated tests included?
* [ ] Code style and in-line documentation are appropriate?
* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

**Comment by evenstensberg** [comment]
Please let me know if you want the github/npm repo for this and I'll transfer it to you

**Comment by evenstensberg** [comment]
https://github.com/evenstensberg/nasa-grammar

**Comment by evenstensberg** [comment]
@unlikelyzero ptal

File: package.json [hunk 1]
```diff
@@ -54,6 +54,7 @@
     "moment": "2.29.4",
     "moment-duration-format": "2.3.2",
     "moment-timezone": "0.5.41",
+    "nasa-grammar": "1.1.1",
```
Comment (depperm) [inline package.json:57]: doesn't this introduce a potential risk like left-pad ([example](https://www.theregister.com/2016/03/23/npm_left_pad_chaos/)) of a random minimal 3rd party library . Couldn't the custom dictionary just be added in the codebase [doc](https://cspell.org/docs/dictionaries-custom/)

Comment (evenstensberg) [reply package.json:57]: @depperm in this case, the npm package is owned by me and I'm happy to hand it over to you. My idea was that we group all nasa repository dictionaries into one package, where each team would have either a namespaced dictionary like `nasa-openmct` and `nasa-ammos`. For now it's just `nasa` but if you would like to use cspell in other js/ts projects, they're under the same umberella. 

The package itself doesn't have any dependencies, so won't need to worry about left-pad

**Comment by ozyx** [comment]
Closing as we will not be taking this change at this time. Thanks!

**Pull request closed without merging**